components/tyk-dev-portal/templates/job-portal.yaml

```
@@ -0,0 +1,56 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: dev-portal-job-{{ include "tyk-dev-portal.fullname" . }}
+  labels:
+    app: {{ include "tyk-dev-portal.fullname" . }}
+    chart: {{ include "tyk-dev-portal.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}  
+spec:
+  template:
+    spec:
+      containers:
+      - name: bootstrap-portal
+        image: curlimages/curl:8.8.0
+        env:
+        - name: ADMIN_EMAIL
+          value: "{{ .Values.global.adminUser.email  }}"
+        - name: ADMIN_PASSWORD
+          {{ if .Values.global.adminUser.useSecretName }}
+          valueFrom:
+            secretKeyRef:
+              name: secrets-{{ include "tyk-dev-portal.fullname" . }}
+              key: adminUserPassword
+          {{ else }}
+          value: {{ .Values.global.adminUser.password | quote }}
+          {{ end }}
+        - name: FIRSTNAME
+          value: "{{ .Values.bootstrapPortal.firstName }}"
+        - name: LASTNAME
+          value: "{{ .Values.bootstrapPortal.lastName }}"                    
+        - name: SERVICE_NAME
+          value: "dev-portal-svc-{{ include "tyk-dev-portal.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local"
```

in same cases (like while using named cluster), `.cluster.local` suffix does not work in internal networking. so, it'd be better if we ignore that
```suggestion
          value: "dev-portal-svc-{{ include "tyk-dev-portal.fullname" . }}.{{ .Release.Namespace }}.svc"
```

---

It works in this case.